### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by faern**
I'm preparing these crates for publishing. So I naturally wanted to shape up the metadata a bit. Here making sure both crates have a readme, and also explaining better how to control the linking to the C library.

Both readmes are simply generated with `cargo readme`.

---
This change is [Reviewable](https://reviewable.io/reviews/mullvad/mnl-rs/8)

### CODE REVIEW

**Review by jvff — CHANGES_REQUESTED** [review]
> Reviewed 4 of 4 files at r1.
> *[Reviewable](https://reviewable.io/reviews/mullvad/mnl-rs/8)* status: all files reviewed, 2 unresolved discussions (waiting on @faern)

___
*[README.md, line 3 at r1](https://reviewable.io/reviews/mullvad/mnl-rs/8#-LL4QgFw65ld8YnEVyRY:-LL4QgFw65ld8YnEVyRZ:b-l4n4in) ([raw file](https://github.com/mullvad/mnl-rs/blob/f8b3d669f5090c97dbc9e50753b927bd65c52713/README.md#L3)):*
> ```Markdown
> # mnl
> 
> Safe abstraction for [`libmnl`]. A minimalistic user-space library oriented to Netlink
> ```

Should this period become a comma to join the two sentences?
___
*[README.md, line 9 at r1](https://reviewable.io/reviews/mullvad/mnl-rs/8#-LL4QsZ12nJ0wcxAM4ap:-LL4QsZ12nJ0wcxAM4aq:b-csvxt5) ([raw file](https://github.com/mullvad/mnl-rs/blob/f8b3d669f5090c97dbc9e50753b927bd65c52713/README.md#L9)):*
> ```Markdown
> to support the parts you need!
> 
> The initial focus here was to support sockets and the parsing of responses. So so far the best
> ```

`so` is repeated here.
___
*[README.md, line 10 at r1](https://reviewable.io/reviews/mullvad/mnl-rs/8#-LL4R0MP5cqGUXIqQqBG:-LL4R0MP5cqGUXIqQqBH:b-5z5tk5) ([raw file](https://github.com/mullvad/mnl-rs/blob/f8b3d669f5090c97dbc9e50753b927bd65c52713/README.md#L10)):*
> ```Markdown
> 
> The initial focus here was to support sockets and the parsing of responses. So so far the best
> covered parts are `mnl_socket_*` and `mnl_cb_run`. However the netlink messages are just
> ```

Slight nitpick, but `best covered` looks a little weird to me. I don't know if it's correct, so feel free to ignore this. I might reword it to `So far the parts that are covered best are [...]`.

**Review by faern — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/mullvad/mnl-rs/8)* status: 3 of 5 files reviewed, 2 unresolved discussions (waiting on @jvff)

___
*[README.md, line 3 at r1](https://reviewable.io/reviews/mullvad/mnl-rs/8#-LL4QgFw65ld8YnEVyRY:-LL4TM3_4NCj2TGjHONC:b-896fix) ([raw file](https://github.com/mullvad/mnl-rs/blob/f8b3d669f5090c97dbc9e50753b927bd65c52713/README.md#L3)):*
<details><summary><i>Previously, jvff (Janito Vaqueiro Ferreira Filho) wrote…</i></summary><blockquote>

Should this period become a comma to join the two sentences?
</blockquote></details>

Done.
___
*[README.md, line 9 at r1](https://reviewable.io/reviews/mullvad/mnl-rs/8#-LL4QsZ12nJ0wcxAM4ap:-LL4TSgh3Crtd6dkU9Pc:b-896fix) ([raw file](https://github.com/mullvad/mnl-rs/blob/f8b3d669f5090c97dbc9e50753b927bd65c52713/README.md#L9)):*
<details><summary><i>Previously, jvff (Janito Vaqueiro Ferreira Filho) wrote…</i></summary><blockquote>

`so` is repeated here.
</blockquote></details>

Done.

**Review by jvff — CHANGES_REQUESTED** [review]
> Reviewed 2 of 2 files at r2, 4 of 4 files at r3.
> *[Reviewable](https://reviewable.io/reviews/mullvad/mnl-rs/8)* status: all files reviewed, 1 unresolved discussion (waiting on @faern)

___
*[mnl-sys/README.md, line 27 at r3](https://reviewable.io/reviews/mullvad/mnl-rs/8#-LL4VCSwDoesJ7Vjaf1c:-LL4VCSwDoesJ7Vjaf1d:b-3uabht) ([raw file](https://github.com/mullvad/mnl-rs/blob/82866b50d05920f9a6c7755e1ff0ced0a90a2e45/mnl-sys/README.md#L27)):*
> ```Markdown
> ```toml
> [dependencies]
> mnl = { version = "0.1", features = ["mnl-1-0-4"] }
> ```

Should this be `mnl-sys = { [...]`?

**Review by faern — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/mullvad/mnl-rs/8)* status: 4 of 6 files reviewed, 1 unresolved discussion (waiting on @jvff and @faern)

___
*[mnl-sys/README.md, line 27 at r3](https://reviewable.io/reviews/mullvad/mnl-rs/8#-LL4VCSwDoesJ7Vjaf1c:-LL4X6YTCeA_qhZKxYld:b8ybrmo) ([raw file](https://github.com/mullvad/mnl-rs/blob/82866b50d05920f9a6c7755e1ff0ced0a90a2e45/mnl-sys/README.md#L27)):*
<details><summary><i>Previously, jvff (Janito Vaqueiro Ferreira Filho) wrote…</i></summary><blockquote>

Should this be `mnl-sys = { [...]`?
</blockquote></details>

Yes :( Fixed

**Review by jvff — APPROVED** [review]
> Reviewed 4 of 4 files at r4.
> *[Reviewable](https://reviewable.io/reviews/mullvad/mnl-rs/8)* status: :shipit: complete! all files reviewed, all discussions resolved

___
*[README.md, line 17 at r4](https://reviewable.io/reviews/mullvad/mnl-rs/8#-LL4XWT738v2ORjw2tIO:-LL4XWT738v2ORjw2tIP:bmolkt5) ([raw file](https://github.com/mullvad/mnl-rs/blob/680830fcf848be423ac931fe0fc526e9dc98b533/README.md#L17)):*
> ```Markdown
> 
> See the documentation for the corresponding sys crate for details: [`mnl-sys`].
> This crate has the same features as the sys crate, so the same features applies here.
> ```

Good idea adding this extra sentence :+1:
